zephyr/module.yml

```diff
@@ -4,3 +4,37 @@ build:
   kconfig-ext: True
   settings:
     dts_root: .
+blobs:
+#IW612
+  - path: iw612/uart_nw61x_se.h
+    sha256: f390b4e2bf157b401310b23742c9a3d05d26ee35821b93811b058dd92019b65e
+    type: img
+    version: '1.0'
+    license-path: zephyr/blobs/license/BSD_3.txt
+    url: https://raw.githubusercontent.com/NXP/wifi_nxp/MCUX_2.15.000/wifi_bt_firmware/nw61x/uart_nw61x_se.h
+    description: "Firmware for NXP IW612 Wi-Fi + Bluetooth single-chip solution"
+    doc-url: https://www.nxp.com/products/wireless-connectivity/wi-fi-plus-bluetooth-plus-802-15-4/2-4-5-ghz-dual-band-1x1-wi-fi-6-802-11ax-plus-bluetooth-5-4-plus-802-15-4-tri-radio-solution:IW612
+  - path: rw61x/rw61x_sb_ble_15d4_combo_a2.bin
+    sha256: e68a45710518b03744c4423af72d3e68ff0506b0f4c4d6ae89b3770907a4c256
+    type: img
+    version: '1.0'
+    license-path: zephyr/blobs/license/LA_OPT_NXP_Software_License.txt
+    url: https://github.com/nxp-mcuxpresso/mcux-sdk/raw/main/components/conn_fwloader/fw_bin/rw61x_sb_ble_15d4_combo_a2.bin
```

Comment:
this one isn't going to work right because it is just grabbing the latest and greatest. The SHA256 won't match on next update.

Reply:
Thanks David, I have updated it to pick a Tag.